src/main/kotlin/com/terraformation/backend/tracking/ObservationService.kt

```diff
@@ -527,9 +549,9 @@ class ObservationService(
   }
 
   /**
-   * Validation rules:
-   * 1. start date can be up to one year from today and not earlier than today
-   * 2. end date should be after the start date but no more than 2 months from the start date
+   * Validation rules: 1a. for non-ad-hoc, start date can be up to one year from today and not
+   * earlier than today. 1b. for ad-hoc, start date can be on or before today.
+   * 2. end date should be after the start date but no more than 2 months from the start date.
```

Comment:
I think the "1a" and "1b" are confusing the formatter because they're not valid Markdown list item numbers.